sui/examples/core_messages/sources/sender.move

```diff
@@ -0,0 +1,190 @@
+/// A simple contracts that demonstrates how to send messages with wormhole.
+module core_messages::sender {
+    use sui::coin::{Self, Coin};
+    use sui::object::{Self, UID};
+    use sui::sui::{SUI};
+    use sui::transfer::{Self};
+    use sui::tx_context::{Self, TxContext};
+
+    use wormhole::state::{State as WormholeState};
+
+    struct DeployerCap has key, store {
+        id: UID
+    }
+
+    struct State has key, store {
+        id: UID,
+        emitter_cap: wormhole::emitter::EmitterCap,
+    }
+
+    fun init(ctx: &mut TxContext) {
+        transfer::transfer(
+            DeployerCap { id: object::new(ctx) },
+            tx_context::sender(ctx)
+        );
+    }
+
+    public entry fun init_with_params(
```

Comment:
the `DeployerCap` pattern is used elsewhere to ensure that only the deployer can initialise the contract with the right parameters (this protects against a malicious DoS attack). However, in the case of this example, the only external parameter used is the reference to the `WormholeState` object, which is globally unique anyway, so I think we should simplify the example by dropping the `DeployerCap` stuff.